src/components/Header/Menu/Menu.tsx

```diff
@@ -1,38 +1,49 @@
 import React, { FC } from 'react'
-import { useTranslation } from 'react-i18next'
+
+import Link from 'next/link'
+import { useRouter } from 'next/router'
 
 import cn from 'classnames'
+import { useTranslation } from 'next-i18next'
 
 import { LanguageName } from 'utils/constants/languages'
 
 import styles from './Menu.module.scss'
 
 type Props = {
-    isActiveMenu?: boolean;
+  isActiveMenuBurger?: boolean;
+  onClickItemMenu?: () => void;
 }
 
 type menuNameType = {
-    id: string,
-    name: string
+  id: string,
+  name: string,
+  path: string
 }
 
-export const Menu: FC<Props> = ({ isActiveMenu }) => {
-    const { t, i18n } = useTranslation()
-
-    const menuName = t('menu', { returnObjects: true }) as menuNameType[]
-
-    return (
-        <div className={cn(styles.wrapper, !isActiveMenu ? styles.wrapperMenu : styles.wrapperActiveMenu)}>
-            <ol className={styles.menuList}>
-                {menuName?.map(item =>
-                    <li
-                        key={item.id}
-                        className={cn(styles.menuItem, { [styles.menuItemUa]: i18n.language === LanguageName.UA })}
-                    >
-                        <a href={`/${item.id}`}>{item.name}</a>
-                    </li>
-                )}
-            </ol>
-        </div>
-    )
+export const Menu: FC<Props> = ({ isActiveMenuBurger, onClickItemMenu }) => {
+  const { t, i18n } = useTranslation()
+
+  const router = useRouter()
+
+  const menuName = t('menu', { returnObjects: true }) as menuNameType[]
+
+  return (
+    <div className={cn(styles.wrapper, !isActiveMenuBurger ? styles.wrapperMenu : styles.wrapperActiveMenuBurger)}>
+      <ul className={styles.menuList}>
+        {Array.isArray(menuName) && menuName.map(item =>
```

Comment:
what does it mean? or set truly  type fro `menuName` or write normal condition

Reply:
fixed